tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -1429,6 +1430,9 @@ TEST_F(EnergyPlusFixture, OutputReportTabular_ZoneMultiplierTest)
 
     ASSERT_TRUE(process_idf(idf_objects));
 
+    // allocate surface level adj ratio data member
+    state->dataHeatBal->SurfWinCoeffAdjRatioIn.dimension(12, 1.0);
+    state->dataHeatBalSurf->SurfWinCoeffAdjRatioOut.dimension(12, 1.0);
     ManageSimulation(*state); // run the design day over the warmup period (24 hrs, 25 days)
```

Comment:
Same here and below. Please check. `SurfWinCoeffAdjRatio`s should be allocated at the beginning of `ManageSimulation` if they are not currently.

Reply:
removed these as well